meta-resin-common/recipes-bsp/u-boot/patches/resin-specific-env-configuration.patch

```diff
@@ -0,0 +1,68 @@
+From 06bd7e9b490575096a7c04c97e3c14d8c12daac8 Mon Sep 17 00:00:00 2001
+From: Andrei Gherzan <andrei@resin.io>
+Date: Wed, 4 Jan 2017 15:58:42 +0100
+Subject: Add resin specific environment configuration
+
+This patch defines all the needed bits on which resinhup relies when swiching
+the root partition. The interface with uboot is done through a file called
+uEnv.txt located in the first partition which, if it exists, might overwrite the
+default root partition index (by default 2). In order to overwrite this,
+uEnv.txt must define `resin_root_part`.
+
+Signed-off-by: Andrei Gherzan <andrei@resin.io>
+
+Upstream-Status: Inappropriate [configuration]
+
+---
+ include/env_resin.h | 40 ++++++++++++++++++++++++++++++++++++++++
+ 1 file changed, 40 insertions(+)
+ create mode 100644 include/env_resin.h
+
+diff --git a/include/env_resin.h b/include/env_resin.h
+new file mode 100644
+index 0000000..ef8a2b0
+--- /dev/null
++++ b/include/env_resin.h
+@@ -0,0 +1,39 @@
++#ifndef HEADER_ENV_RESIN_H
++#define HEADER_ENV_RESIN_H
++
++/*
++ * Assumes defined:
```

Comment:
So if they are assumed in the env they should be coming from an uEnv.txt file, right?